src/lava/lib/dnf/connect/connect.py

```diff
@@ -195,7 +201,9 @@ def _compute_weights(ops: ty.List[AbstractOperation]) -> np.ndarray:
 
 def _make_connections(src_op: OutPort,
                       dst_ip: InPort,
-                      weights: np.ndarray) -> AbstractProcess:
+                      weights: np.ndarray,
+                      connection: ty.Optional[ty.Union[Sparse, Dense]] = None
```

Comment:
The parameter name and type annotation should be the same as in the method above.